app.js

```diff
@@ -0,0 +1,95 @@
+/* eslint-disable max-classes-per-file */
+
+class Book {
```

Comment:
- [ ] Let us try to fix eslint without disabling line of code, You have used many classes in one, please find another way of correcting this issue.

Reply:
Hi..please note that it is OK to have multiple classes for this milestone. Kindly find attached a screenshot of  the project requirements
![code-review](https://user-images.githubusercontent.com/33420420/133427540-12ea99dc-a97a-4c68-bc01-8b491f385f67.png)